windowslib/src/main/scala/scala/scalanative/windows/util/HelperMethods.scala

```diff
@@ -0,0 +1,56 @@
+package scala.scalanative.windows.util
+
+import java.io.IOException
+import scala.scalanative.unsafe._
+import scala.scalanative.windows.HandleApi._
+import scala.scalanative.windows.HandleApiExt._
+import scala.scalanative.windows.FileApiExt._
+import scala.scalanative.windows.ProcessThreadsApi._
+import scala.scalanative.windows._
+
+object HelperMethods {
```

Comment:
```suggestion
private[windows] object HelperMethods {
```

Reply:
Actually, this should be moved to javalib and made `private[java]` as discussed in the previous PR. It was not moved after rebasing.